torchbenchmark/models/yolov3/__init__.py

```diff
@@ -52,9 +52,24 @@ def get_module(self):
         opt.cfg = check_file(opt.cfg)  # check file
         opt.names = check_file(opt.names)  # check file
         model = Darknet(opt.cfg, opt.img_size)
-        model.to(opt.device).eval()
+        model.to(opt.device)
         input = (torch.rand(1, 3, 384, 512).to(opt.device),)
         return model, input
+
+    def set_eval(self):
+        self.set_mode(False)
+
+    def set_train(self):
+        self.set_mode(True)
+
+    def set_mode(self, train):
+        if not train:
+            (model, _) = self.get_module()
```

Comment:
in this case it would be fine to just no-op set_mode and rely on the existing line55 model.to(opt.device).eval() right?

Reply:
I find it less consistent and more confusing, but I don't have a strong opinion here )

Reply:
i also don't have a strong opinion.